docs/product-getting-started-guide/src/main/asciidoc/data-model-proc.adoc

```diff
@@ -46,7 +46,7 @@ image::field-data-applicant.png[]
 == Viewing the Property data object
```

Comment:
I don't see it in this PR, but search for "Review the Property data object" and "Review the Application data object" bold **Property** and **Application**. Comes up a couple of times. 

Also, it's not in this PR, but search "rules developer, you define rules" and change to "you can define" (in Business rules and GDT section). There are many ways to define rules. GDTs are just one of the ways. This reads like it's the only way.

Also, the first paragraph in the Business rules and GDT section should be about guided rules, not the GDT. The second paragraph is about GDT. Use the description of guided rules in my guided-rules assembly if you like. Right now it's like the two paragraphs about about GDTs, and then we go immediately into creating guided rules, so it's confusing. Didn't notice this last time.